tests/test_scripts.py

```diff
@@ -611,7 +620,7 @@ def records(self, lane, annotator, mirror_integration, start_time=None):
 
         eq_(lane, exporter.called_with[0][0])
         assert isinstance(exporter.called_with[0][1], MARCLibraryAnnotator)
-        eq_(exporter.integration, exporter.called_with[0][2])
+        eq_(the_linked_integration, exporter.called_with[0][2])
```

Comment:
This is exactly what I was looking for -- verifying that `the_linked_integration` is passed in rather than `integration`.